tango_simlib/tests/test_sim_xmi_parser.py

```diff
@@ -310,6 +311,7 @@ def test_attribute_list(self):
             "Actual tango device attribute list differs from expected " "list!",
         )
 
+    @unittest.skipIf((not future.utils.PY2), "Only for Python 2")
```

Comment:
Don't we need to attempt these for Python3 as well? (assuming that is how this decorator works)

Reply:
@lanceWilliams  not sure we need input from @kmadisa , @ajoubertza, @samueltwum1  
the description contains a unicode character that is replaced with a `?` and this is present in the `expected_pressure_attr_info` https://github.com/ska-sa/tango-simlib/blob/9b9ccb7df9477d92fe3b8d1ae2a5b12c79fe2ab0/tango_simlib/tests/test_sim_xmi_parser.py#L120
When this test is executed in Python3 the correct unicode value is returned and therefor I skipped the tests and the functionality is different in Python2 and Python3 . 

Perhaps the test should not be skipped and the ascii encoded value should be returned and not unicode in Python3 also?